### PR TITLE
MOTS-63: Initialize location dropdowns on CHW edit form

### PR DESCRIPTION
**Pull request by pmuchowski**

### CODE REVIEW

File: src/webapp/js/components/health-workers-form.js [hunk 1]
```diff
@@ -8,9 +8,8 @@ import DateTime from 'react-datetime';
 import 'react-datetime/css/react-datetime.css';
 
 import FormField from './form-field';
-import { createHealthWorker } from '../actions';
-import { clearFields, getAttributesForObjectSelect } from '../utils/form-utils';
-import apiClient from '../utils/api-client';
+import { createHealthWorker, fetchLocations } from '../actions';
```
Comment (dpiaseckisoldevelo) [inline src/webapp/js/components/health-workers-form.js]: I think you can delete createHealthWorker from here. Please let me know.

File: src/webapp/js/components/health-workers-form.js [hunk 2]
```diff
@@ -239,52 +212,35 @@ const selector = formValueSelector(CHW_FORM_NAME);
...
-  name: PropTypes.string.isRequired,
-  facilities: PropTypes.arrayOf(facilityPropType),
-});
-
-const districtPropType = PropTypes.shape({
-  id: PropTypes.string.isRequired,
-  name: PropTypes.string.isRequired,
-  chiefdoms: PropTypes.arrayOf(chiefdomPropType),
-});
+})(connect(mapStateToProps, { createHealthWorker, fetchLocations })(HealthWorkersForm));
```
Comment (dpiaseckisoldevelo) [inline src/webapp/js/components/health-workers-form.js]: Same here for createHealthWorker

File: src/webapp/js/components/health-workers-form.js [hunk 3]
```diff
@@ -174,24 +160,11 @@ class HealthWorkersForm extends Component {
...
-  }
-
-  fetchLocations() {
-    const url = '/api/districts';
-
-    apiClient.get(url).then((response) => {
-      const availableLocations = response.data;
-      this.setState({ availableLocations });
-    });
+    this.props.fetchLocations();
```
Comment (dpiaseckisoldevelo) [inline src/webapp/js/components/health-workers-form.js]: Should we fetch locations on every load of Health Worker form? Maybe we can find some outer component, to make some initial actions there. If locations don't change a lot. What do you think?

File: src/webapp/js/reducers/locations_reducer.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+import { FETCH_LOCATIONS } from '../actions/types';
+
+export default function (state = {}, action) {
```
Comment (dpiaseckisoldevelo) [inline src/webapp/js/reducers/locations_reducer.js]: Will we use locations in some other places in the app? Are there in plans some other operations on locations or some api calls with locations? I'm just curious, if the name for reducer is good and reducer is necessary.

File: src/webapp/js/components/health-workers-form.js (hunk 1 above)
Comment (pmuchowski) [reply src/webapp/js/components/health-workers-form.js]: yes, you're right, I forgot to remove it during previous refactor

File: src/webapp/js/components/health-workers-form.js (hunk 3 above)
Comment (pmuchowski) [reply src/webapp/js/components/health-workers-form.js]: I was thinking about this, we can talk more tomorrow on standup

File: src/webapp/js/reducers/locations_reducer.js (hunk 4 above)
Comment (pmuchowski) [reply src/webapp/js/reducers/locations_reducer.js]: I think we will need locations on module assignment screen

**Pull request merged**